webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/appconfig/ConfigService.scala

```diff
@@ -224,6 +224,8 @@ trait ReadConfigService {
 
   def rudder_generation_continue_on_error(): IOResult[Boolean]
 
+  def rudder_setup_done(): IOResult[Boolean]
```

Comment:
I think it should no be a boolean are, but a (at least) three case: 
- not done
- remind me latter
- done

Reply:
(with remind me storing the iso-date at which we should show again the pop-up)